libbeat/publisher/queue/memqueue/ackloop.go

```diff
@@ -53,13 +53,20 @@ func (l *ackLoop) run() {
 			count, events := lst.count()
 			l.lst.concat(&lst)
 
-			// log.Debugf("ackloop: scheduledACKs count=%v events=%v\n", count, events)
+			// log.Debug("ACK List:")
```

Comment:
Is this commented out on purpose?

Reply:
Yes. debug creates quite some noise if not required. There are a many debug statements in this package being commented out and only selectively uncommented when required (when debugging locally).